### PR TITLE
Adds Willow Garage data set.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

**Comment by damonkohler** [comment]
FYI @macmason

File: docs/source/data.rst [hunk 1]
```diff
@@ -103,3 +105,124 @@ SLAM, and data from an additional vertical (i.e. push broom) LIDAR.
...
+===========================================
+
+3D data sets are still being curated and will be made available soon.
+
+PR2 - Willow Garage
+===================
+
+This is the Willow Garage data set, described in:
+
+"An Object-Based Semantic World Model for Long-Term Change Detection and
```
Comment (SirVer) [inline docs/source/data.rst]: add an itemization * here too

Comment (damonkohler) [inline docs/source/data.rst]: Doen.

File: docs/source/data.rst [hunk 2]
```diff
@@ -103,3 +105,124 @@ SLAM, and data from an additional vertical (i.e. push broom) LIDAR.
 `b2-2016-04-27-12-31-41.bag <https://storage.googleapis.com/cartographer-public-data/bags/backpack_2d/b2-2016-04-27-12-31-41.bag>`_
 =================================================================================================================================== ==========================================
 
+3D Cartographer Backpack - Deutsches Museum
```
Comment (wohe) [inline docs/source/data.rst]: Should this use an en-dash, i.e., "–"? Same in other places.

File: docs/source/data.rst [hunk 3]
```diff
@@ -103,3 +105,124 @@ SLAM, and data from an additional vertical (i.e. push broom) LIDAR.
...
+===========================================
+
+3D data sets are still being curated and will be made available soon.
+
+PR2 - Willow Garage
+===================
+
+This is the Willow Garage data set, described in:
+
+  * "An Object-Based Semantic World Model for Long-Term Change Detection and
```
Comment (wohe) [inline docs/source/data.rst]: Unindent?

File: docs/source/data.rst (hunk 2 above)
Comment (damonkohler) [inline docs/source/data.rst]: Done.

File: docs/source/data.rst (hunk 3 above)
Comment (damonkohler) [inline docs/source/data.rst]: Done.

**Pull request merged**